core/iwasm/libraries/libc-builtin/libc_builtin_wrapper.c

```diff
@@ -473,12 +473,16 @@ snprintf_wrapper(wasm_exec_env_t exec_env, char *str, uint32 size,
 static int
 puts_wrapper(wasm_exec_env_t exec_env, const char *str)
 {
+    (void)exec_env;
```

Comment:
i feel it's cleaner to use attribute((unused)) to suppress the warning. how do you think?

Reply:
It may be good, but I found that other project like zephyr also uses `(void)xxx`, and it define macro `ARG_UNUSED(x)`:
https://github.com/zephyrproject-rtos/zephyr/blob/b302308e5f666a960a84ff8d76cfbe531a932fdf/include/zephyr/toolchain/gcc.h#L263

Maybe we can also define a similar macro?